packages/crowbook/build.sh

```diff
@@ -2,29 +2,13 @@ TERMUX_PKG_HOMEPAGE=https://github.com/crowdagger/crowbook
 TERMUX_PKG_DESCRIPTION="Allows you to write a book in Markdown without worrying about formatting or typography"
 TERMUX_PKG_LICENSE="LGPL-2.1"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.16.1"
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_VERSION="0.17.0"
 TERMUX_PKG_SRCURL=https://github.com/crowdagger/crowbook/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=2e49a10f1b14666d4f740e9a22a588d44b137c3fca0932afc50ded0280450311
+TERMUX_PKG_SHA256=13c628722fedf1bfebc0fd334e9ffa41a62888e3195d2ad1e5ea851694dc4a4c
+TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="openssl"
 TERMUX_PKG_BUILD_IN_SRC=true
 
-termux_step_pre_configure() {
-	termux_setup_rust
-
-	: "${CARGO_HOME:=$HOME/.cargo}"
-	export CARGO_HOME
-
-	cargo vendor
```

Comment:
Please also remove `termux_step_post_make_install`.

Reply:
OK, I'll do it in next commit.

Reply:
Done fc6129e68ab37befce889b543a54fce751d2eed6